pilot/cmd/pilot-agent/main.go

```diff
@@ -134,8 +135,9 @@ var (
 		"the provider of Pilot DNS certificate.").Get()
 	jwtPolicy = env.RegisterStringVar("JWT_POLICY", jwt.JWTPolicyThirdPartyJWT,
 		"The JWT validation policy.")
-	outputKeyCertToDir = env.RegisterStringVar("OUTPUT_KEY_CERT_TO_DIRECTORY", "",
-		"The output directory for the key and certificate. If empty, no output of key and certificate.").Get()
+	outputKeyCertToDir = env.RegisterStringVar("OUTPUT_CERTS", "",
```

Comment:
This change will cause the problem in https://github.com/istio/istio/issues/21843.

Reply:
Will update the prom templates as well. No tests failing on that ?

Reply:
Done

Reply:
Can you approve ? Trying to get it in...

Reply:
Approved.